src/mina-signer/consistency.unit-test.ts

```diff
@@ -178,6 +193,49 @@ let fullCommitment = hashWithPrefix(prefixes.accountUpdateCons, [
 ]);
 expect(fullCommitment).toEqual(ocamlCommitments.fullCommitment.toBigInt());
 
+// private key to/from base58
+let feePayerKey = PrivateKey.fromBase58(feePayerKeyBase58);
+
+let feePayerCompressed = ScalarSnarky.toFieldsCompressed(feePayerKeySnarky.s);
+expect(feePayerKey).toEqual(feePayerCompressed.field.toBigInt());
+expect(PrivateKey.toBase58(feePayerKey)).toEqual(feePayerKeyBase58);
+
+// signature
+let signature = signFieldElement(fullCommitment, feePayerKey, 'testnet');
+let signatureBase58 = Signature.toBase58(signature);
+let signatureOcaml = Ledger.signFieldElement(
+  ocamlCommitments.fullCommitment,
+  feePayerKeySnarky
+);
+expect(signatureBase58).toEqual(signatureOcaml);
+
+let isValid = verifyFieldElement(
```

Comment:
Would be good to have at least one valid `mainnet` testcase

Reply:
Good point! I exposed them from OCaml now: https://github.com/MinaProtocol/mina/pull/12337